website/docs/integrations/amplitude.mdx

```diff
@@ -260,8 +260,8 @@ Check your Experiments page in Amplitude and select your feature flag as the Exp
 ### Usage with custom chart
 If you don't have access to the Experiments feature in Amplitude, you can create a custom chart based on the `Exposure` event. 
 You can filter for your feature flag keys with the `Flag Key` property and visualize the different variants by using the `Variant` property as a Breakdown. Example:
-<img src="/docs/assets/amplitude/customchart.png" className="bordered-img zoomable" alt="Amplitude custom chart" />
+<img src="/docs/assets/integrations/amplitude/customchart.png" className="bordered-img zoomable" alt="Amplitude custom chart" decoding="async" loading="lazy"/>
```

Comment:
The src URL has changed.

Is this intentional?

Reply:
fixed it